Backend/UnhousedOutreach.Api/Controllers/LookupsController.cs

```diff
@@ -140,15 +140,15 @@ public async Task SetPantsSize([FromBody]KeyValuePair<int?, string> pantsSize, [
     }
     #endregion
 
-    #region Need
-    [HttpGet("need")]
-    public async Task<Dictionary<int, string>> GetNeedLookup([FromQuery(Name = "otid")]int outreachTeamId)
+    #region Needs
+    [HttpGet("needs")]
+    public async Task<Dictionary<int, string>> GetNeedsLookup([FromQuery(Name = "otid")]int outreachTeamId)
     {
         return await repository.GetNeeds(outreachTeamId);
     }
 
-    [HttpPost("need")]
-    public async Task SetNeed([FromBody]KeyValuePair<int?, string> need, [FromQuery(Name = "otid")]int outreachTeamId)
+    [HttpPost("needs")]
+    public async Task SetNeeds([FromBody]KeyValuePair<int?, string> need, [FromQuery(Name = "otid")]int outreachTeamId)
```

Comment:
This (and above line) should not have been made plural. This sets ONE need

Reply:
I believe I have addressed this issue.